### PR TITLE
Add hover effect to favorite-star in events

### PR DESCRIPTION
**Pull request by ingraso**

### CODE REVIEW

File: app/routes/events/components/EventDetail/EventDetail.css [hunk 1]
```diff
@@ -4,6 +4,15 @@
   color: var(--color-event-red);
   margin-right: 10px;
 }
+/*
```
Comment (SmithPeder) [inline app/routes/events/components/EventDetail/EventDetail.css]: If nobody uses this anymore we can just remove it.

File: package.json [hunk 2]
```diff
@@ -135,7 +135,7 @@
     "postcss-import": "^11.0.0",
     "postcss-loader": "^2.0.6",
     "postcss-nested": "^2.1.0",
-    "prettier": "^1.14.3",
+    "prettier": "1.14.3",
```
Comment (SmithPeder) [inline package.json]: Any specific reason for this? 🥇

File: app/routes/events/components/EventDetail/index.js [hunk 3]
```diff
@@ -35,7 +36,20 @@ type InterestedButtonProps = {
...
 const InterestedButton = ({ isInterested }: InterestedButtonProps) => {
   const icon = isInterested ? 'star' : 'star-outline';
-  return <Icon className={styles.star} name={icon} />;
+  return (
+    <div style={{ display: 'inline-block' }}>
+      <Tooltip
+        style={{ lineHeight: '1.3rem' }}
+        content={
+          <span style={{ fontSize: '1rem', fontWeight: '100', padding: '0' }}>
+            Favoritt
```
Comment (JonasBak) [inline app/routes/events/components/EventDetail/index.js]: Maybe write something about what happens when you follow an event? (It shows up in your calendar)

File: app/routes/events/components/EventDetail/index.js [hunk 4]
```diff
@@ -35,7 +36,20 @@ type InterestedButtonProps = {
 
 const InterestedButton = ({ isInterested }: InterestedButtonProps) => {
   const icon = isInterested ? 'star' : 'star-outline';
-  return <Icon className={styles.star} name={icon} />;
+  return (
+    <div style={{ display: 'inline-block' }}>
+      <Tooltip
+        style={{ lineHeight: '1.3rem' }}
+        content={
+          <span style={{ fontSize: '1rem', fontWeight: '100', padding: '0' }}>
```
Comment (JonasBak) [inline app/routes/events/components/EventDetail/index.js]: I don't think @SmithPeder will be too pleased with inline styles

**Pull request merged**